Sources/Components/HappinessRating/HappinessRatingView.swift

```diff
@@ -6,13 +6,21 @@ import UIKit
 
 public protocol HappinessRatingViewDelegate: AnyObject {
     func happinessRatingView(_ happinessRatingView: HappinessRatingView, didSelectRating rating: HappinessRating)
+
+    // MARK: - Optional
+    func happinessRatingView(_ happinessRatingView: HappinessRatingView, textFor: HappinessRating) -> String?
```

Comment:
This method doesn't seem to be called from anywhere🤔

Reply:
Optional implementation so we don't force it to be implemented everywhere.

Reply:
Never mind... herp derp